depscan/lib/analysis.py

```diff
@@ -840,6 +900,86 @@ def prepare_vdr(options: PrepareVdrOptions):
     return pkg_vulnerabilities, pkg_group_rows
 
 
+def get_version_range(package_issue, purl):
+    new_prop = {}
+    if (affected_location := package_issue.get("affected_location")) and (
+            affected_version := affected_location.get("version")):
+        try:
+            ppurl = PackageURL.from_string(purl)
+            new_prop = {
+                "name": "affectedVersionRange",
+                "value": f'{ppurl.name}@'
+                         f'{affected_version}'
+            }
+            if ppurl.namespace:
+                new_prop["value"] = f'{ppurl.namespace}/{new_prop["value"]}'
+        except ValueError:
+            ppurl = purl.split("@")
+            if len(ppurl) == 2:
+                new_prop = {
+                    "name": "affectedVersionRange",
+                    "value": f'{ppurl[0]}@{affected_version}'
+                }
+
+    return new_prop
+
+
+def cvss_to_vdr(res):
```

Comment:
@prabhu I can check for mandatory fields using cvss, but it doesn't make sense to move to vdb and I wouldn't want to add the dependency for such a minimal use. I say it doesn't make sense to move this to vdb because the purpose of the function is exclusive to depscan.